statsmodels/iolib/summary2.py

```diff
@@ -1,14 +1,16 @@
+import datetime
+import re
+import textwrap
```

Comment:
+1 for the stdlib --> 3rd party --> 1st party import ordering

Reply:
It is correct except it should be compat, std lib, 3rd, 1st.